assets/out

```diff
@@ -137,6 +137,9 @@ helm_install() {
     helm_echo="$helm_echo --devel"
   fi
   if [ -n "$version" ]; then
+    if [ -f "$source/$version" ]; then
```

Comment:
I realize we have not been super consistent, but for most stuff that gets read from files we use a "_file" suffix. So I suggest you read the filename from "version_file" out of the json (same as release_file).

Reply:
I think that fundamentally, this is a concourse resource and concourse resource inputs are files, it should just be able to accept either a string or a file.  I don't like the idea of having to know that to pass `--version` to helm you have to know to do two different things depending on the input.  Hence the "is it a file, parse it, if not, pass it".

As it happens we have more requirements that don't really fit the design of this module (we need to decrypt ansible-vault files) so we'll be running from our fork anyway, so feel free to take the idea and implement it how you feel is right for your project.